geotools-suite/pom.xml

```diff
@@ -17,13 +17,13 @@
 
 	<groupId>org.codice.thirdparty</groupId>
 	<artifactId>geotools-suite</artifactId>
-	<version>19.1_2</version>
+	<version>20.1_2</version>
```

Comment:
❗️ The bundle version should go to 20.1_1. If we need to make any further changes to this pom then we'd bump it to 20.1_2 and so on.